packages/core/src/components/meter/meter.scss

```diff
@@ -0,0 +1,86 @@
+:host {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  width: 100%;
+  word-wrap: break-word;
+}
+
+.container-text {
+  align-items: flex-end;
+  display: flex;
+  font: var(--text-body-small);
+  font-weight: 300;
```

Comment:
block: `var(--text-body-small)` is not bold, so we don't need it

Reply:
Made: https://github.com/juntossomosmais/atomium/pull/695/commits/b6d3528d99946d7bc57574d29acef6fc000b8c5c